Common subdirectories: ivy-ocaml-1.3.2.orig/debian and ivy-ocaml-1.3.2/debian
Common subdirectories: ivy-ocaml-1.3.2.orig/examples and ivy-ocaml-1.3.2/examples
Common subdirectories: ivy-ocaml-1.3.2.orig/.git and ivy-ocaml-1.3.2/.git
diff -u ivy-ocaml-1.3.2.orig/Makefile ivy-ocaml-1.3.2/Makefile
--- ivy-ocaml-1.3.2.orig/Makefile	2021-05-07 14:11:27.786761172 +0200
+++ ivy-ocaml-1.3.2/Makefile	2021-05-07 14:40:28.787120375 +0200
@@ -50,14 +50,6 @@
 IVYGLIB_CLIBS = -lglibivy -lglib-2.0
 endif
 
-IVYTCL_CLIBS=$(shell pkg-config --libs ivy-tcl)
-ifeq ($(strip $(IVYTCL_CLIBS)),)
-IVYTCL_CLIBS = -ltclivy
-endif
-
-# at least on Debian this is a symlink to the latest tcl version if tcl-dev is installed
-TKINC=-I/usr/include/tcl
-
 # by default use fPIC on all systems
 FPIC ?= -fPIC
 
@@ -82,21 +74,11 @@
 GLIBIVYMLI= $(GLIBIVY:.ml=.mli)
 GLIBIVYCMX= $(GLIBIVY:.ml=.cmx)
 
-TKIVY = ivy.ml tkIvy.ml
-
-TKIVYCMO= $(TKIVY:.ml=.cmo)
-TKIVYCMI= $(TKIVY:.ml=.cmi)
-TKIVYMLI= $(TKIVY:.ml=.mli)
-TKIVYCMX= $(TKIVY:.ml=.cmx)
-
-
 IVYLIBS = ivy-ocaml.cma ivy-ocaml.cmxa
 GLIBIVYLIBS = glibivy-ocaml.cma glibivy-ocaml.cmxa
-TKIVYLIBS = tkivy-ocaml.cma tkivy-ocaml.cmxa
 
 IVYSTATIC = libivy-ocaml.a ivy-ocaml.a
 GLIBIVYSTATIC = libglibivy-ocaml.a glibivy-ocaml.a
-TKIVYSTATIC = libtkivy-ocaml.a tkivy-ocaml.a
 LIBS = ivy-ocaml.cma glibivy-ocaml.cma
 XLIBS = ivy-ocaml.cmxa glibivy-ocaml.cmxa
 
@@ -108,20 +90,16 @@
 
 ivy : $(IVYLIBS)
 glibivy : $(GLIBIVYLIBS)
-tkivy : $(TKIVYLIBS)
 
 IVY_ALL_LIBS = $(IVYLIBS) $(IVYSTATIC) dllivy-ocaml.so
 GLIBIVY_ALL_LIBS = $(GLIBIVYLIBS) $(GLIBIVYSTATIC) dllglibivy-ocaml.so
-TKIVY_ALL_LIBS = $(TKIVYLIBS) $(TKIVYSTATIC) dlltkivy-ocaml.so
 
 IVY_INST_FILES = $(IVYMLI) $(IVYCMI) $(IVYCMX) $(IVY_ALL_LIBS)
 GLIBIVY_INST_FILES = $(GLIBIVYMLI) $(GLIBIVYCMI) $(GLIBIVYCMX) $(GLIBIVY_ALL_LIBS)
-TKIVY_INST_FILES = $(TKIVYMLI) $(TKIVYCMI) $(TKIVYCMX) $(TKIVY_ALL_LIBS)
 
-install : $(IVY_INST_FILES) $(GLIBIVY_INST_FILES) $(TKIVY_INST_FILES)
+install : $(IVY_INST_FILES) $(GLIBIVY_INST_FILES)
 	mv META.ivy META && ocamlfind install $(OCAMLFINDFLAGS) ivy META $(IVY_INST_FILES) && mv META META.ivy || (mv META META.ivy && exit 1)
 	mv META.glibivy META && ocamlfind install $(OCAMLFINDFLAGS) glibivy META $(GLIBIVY_INST_FILES) && mv META META.glibivy || (mv META META.glibivy && exit 1)
-	mv META.tkivy META && ocamlfind install $(OCAMLFINDFLAGS) tkivy META $(TKIVY_INST_FILES) && mv META META.tkivy || (mv META META.tkivy && exit 1)
 ifeq ($(COMPAT_SYMLINK_CREATE), y)
 	# make some symlinks for backwards compatibility
 	@echo "Creating symlinks for backwards compatibility..."
@@ -149,13 +127,6 @@
 glibivy-ocaml.cmxa : $(GLIBIVYCMX) civy.o cglibivy.o
 	$(OCAMLMKLIB) -o glibivy-ocaml $^ $(LIBRARYS) $(IVYGLIB_CLIBS) -lpcre
 
-tkivy-ocaml.cma : $(TKIVYCMO) civy.o ctkivy.o
-	$(OCAMLMKLIB) -o tkivy-ocaml $^ $(LIBRARYS) $(IVYTCL_CLIBS)
-
-tkivy-ocaml.cmxa : $(TKIVYCMX) civy.o ctkivy.o
-	$(OCAMLMKLIB) -o tkivy-ocaml $^ $(LIBRARYS) $(IVYTCL_CLIBS)
-
-
 .SUFFIXES:
 .SUFFIXES: .ml .mli .mly .mll .cmi .cmo .cmx .c .o .out .opt
 
@@ -179,7 +150,7 @@
 clean:
 	\rm -fr *.cm* *.o *.a .depend *~ *.out *.opt .depend *.so *-stamp debian/ivy-ocaml debian/files debian/ivy-ocaml.debhelper.log debian/ivy-ocaml.substvars debian/*~
 
-.PHONY: all dev ivy glibivy tkivy install uninstall clean
+.PHONY: all dev ivy glibivy install uninstall clean
 
 .depend:
 	$(OCAMLDEP) $(INCLUDES) *.mli *.ml > .depend
